images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile_metering.yml

```diff
@@ -11,3 +11,6 @@
       - template_file: templates/metering/metering-rolebindings.yaml
         apis: [ {kind: rolebindings, api_version: 'rbac.authorization.k8s.io/v1'} ]
         prune_label_value: openshift-metering-rolebindings
+      - template_file: templates/metering/metering-root-ca-secret.yaml
+        apis: [ {kind: Secret} ]
+        prune_label_value: openshift-metering-root-ca-secret
```

Comment:
maybe we want `create: "{{ meteringconfig_enable_tls }}"` since we only really need this if we're generating the cert right?

Reply:
I was under the impression that we were always going to create this after our discussion this morning. I had the `create: ` value conditioned on `meteringconfig_enable_tls` initially. 

No problem, I can update this.

Reply:
Well I was trying to think what we would put into it when tls is disabled, and it seems like nothing, so that's why I was thinking this.